TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/EntityArray.java

```diff
@@ -0,0 +1,69 @@
+package com.ybsystem.tweetmate.models.entities;
+
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.libs.propertychange.PropertyChangeListener;
+import com.ybsystem.tweetmate.libs.propertychange.PropertyChangeSupport;
+import com.ybsystem.tweetmate.databases.PrefEntity;
+import com.ybsystem.tweetmate.utils.ToastUtils;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+
+import static com.ybsystem.tweetmate.resources.ResString.STR_FAIL_NO_MORE_MOVE;
+
+public class EntityArray<T> extends ArrayList<T>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS2160)